docs/migration-v8.md

```diff
@@ -186,9 +186,9 @@ ResiliencePipeline pipeline = new ResiliencePipelineBuilder()
 <!-- endSnippet -->
 
 > [!IMPORTANT]
-> In v7, the policy wrap ordering is different; the policy added first was executed last. In v8, the execution order matches the order in which they were added.
+> In v7, the policy wrap ordering is different; the policy added first was executed last (FILO). In v8, the execution order matches the order in which they were added (FIFO).
 
-## Migrating retry policy
+## Migrating Retry Policies
```

Comment:
Any reason to capitalize the headings here?

In-general, only first word is capitalized.

Reply:
I changed them to Title Case as I thought headings were to be considered as titles.

Reply:
I think it's better to leave them as before, for consistency and alignment with official docs guidelines.

Reply:
Ok I'll change them back. Do you have a link to these guidelines so I can take a look?

Reply:
https://learn.microsoft.com/en-us/contribute/content/dotnet/dotnet-style-guide#headings